### PR TITLE
[fix #180] add Pull Request template

### LINKED ISSUE #180: Use Pull Request template to ensure that only Rmd file gets submitted
I'm wondering if we could use the new PR templating system that GItHub now provides (https://github.com/blog/2111-issue-and-pull-request-templates) to indicate that only Rmd files should be (normally) submitted when editing one of the lessons.

### PR DESCRIPTION
**Pull request by fmichonneau**
Here is a first stab at #180. Let me know if I should add anything else.

### CODE REVIEW

File: PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+Thank you for pull request! If you are proposing a change to one of the lesson,
```
Comment (hdashnow) [inline PULL_REQUEST_TEMPLATE.md]: Suggestion:
Thank you **for submitting a** pull request! If you are proposing a change to one of the **lessons**,

**Comment by hdashnow** [comment]
Thanks @fmichonneau! I'm guessing I can only test this by accepting the PR and observing the change in GitHub behaviour?

Could you please fix that first sentence before I merge?

**Comment by fmichonneau** [comment]
Thank @hdashnow. I just committed the requested changes. I'll be curious to see how it works out.

**Comment by hdashnow** [comment]
@fmichonneau it works! Thank you.

**Pull request merged**